index.js

```diff
@@ -1,13 +1,222 @@
 const body = document.querySelector('body');
 const hamburger = document.querySelector('.hamburger');
 const close = document.querySelector('.close');
-
 const menu = document.querySelector('.hide_mobile');
+const projectCards = document.querySelector('.wrapper');
+const contactContain = document.getElementById('contact');
+const aboutContain = document.getElementById('aboutme');
+const mainContain = document.getElementById('portfolio');
+/* const popupContain = document.querySelectorAll('popup-window'); */
+const homepageContain = document.getElementById('homepage');
+// const recentworkContain = document.querySelector("recent-work");
+
```

Comment:
- [ ] [OPTIONAL] Remove all your comments ( // ) in your JavaScript file to make your code more impressive 😉